microservices/api/src/controllers/ConfigurationController.php

```diff
@@ -5,14 +5,13 @@
 namespace herams\api\controllers;
 
 use herams\common\helpers\ConfigurationProvider;
-use League\ISO3166\ISO3166;
 use yii\web\Response;
 
 class ConfigurationController extends Controller
 {
     public function actionCountries(
         ConfigurationProvider $configurationProvider,
-        Response $response): array
+        Response              $response): array
```

Comment:
Don't do manual indenting like this, we use a tool for this : `vendor/bin/ecs` -- easy coding standard.